dolphinscheduler-ui/src/views/projects/task/components/node/tasks/use-dinky.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {reactive} from 'vue'
+import * as Fields from '../fields/index'
+import type {IJsonItem, INodeData, ITaskData} from '../types'
+
+export function useDinky({
+                             projectCode,
+                             from = 0,
+                             readonly,
+                             data
+                         }: {
+    projectCode: number
+    from?: number
+    readonly?: boolean
+    data?: ITaskData
+}) {
+    const model = reactive({
+        name: '',
+        taskType: 'DINKY',
+        flag: 'YES',
+        description: '',
+        timeoutFlag: false,
+        localParams: [],
+        environmentCode: null,
+        failRetryInterval: 1,
+        failRetryTimes: 0,
+        workerGroup: 'default',
+        delayTime: 0,
+        timeout: 30
```

Comment:
Here should set the timeoutNotifyStrategy default value to ['WARN'].
plx check this #10259 .

Reply:
Hi. I have changed the code.